core/src/main/java/com/google/errorprone/bugpatterns/StringSplit.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 Two Sigma Open Source. All Rights Reserved.
```

Comment:
This should be

```
Copyright 2017 The Error Prone Authors.
```

You're welcome to add Two Sigma to https://github.com/google/error-prone/blob/master/AUTHORS. (There's some documentation on copyright and AUTHORS files here: https://opensource.google.com/docs/releasing/authors/)